boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z280_rtu0_defconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_XIP=n
+CONFIG_ISR_STACK_SIZE=512
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=8000000
+CONFIG_ARM_MPU=y
+CONFIG_PINCTRL=y
```

Comment:
please, do not enable pinctrl at board level, let drivers who need it select the option.

Reply:
Hello, i can remove this at here. But looking around in-tree boards, various boards have this Kconfig option enabled. So I'm wondering if we have this guideline documented somewhere for people who introduce new boards to be aware?